### PR TITLE
Update JavaDocs

### PR DESCRIPTION
**Pull request by pjanthony2001**

### CODE REVIEW

**Review by Jolonauh — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/RedoCommand.java [hunk 1]
```diff
@@ -8,12 +8,11 @@
 import seedu.address.model.Model;
 
 /**
- * Undoes a command
+ * Redoes a command, reverting the model to that state
```
Comment (Rishit02) [inline src/main/java/seedu/address/logic/commands/RedoCommand.java:11]: Can add a full stop.

File: src/main/java/seedu/address/model/ReadOnlyAddressBook.java [hunk 2]
```diff
@@ -14,5 +14,9 @@ public interface ReadOnlyAddressBook {
      * This list will not contain any duplicate persons.
      */
     ObservableList<Person> getPersonList();
+
+    /**
+     * Returns an unmodifiable deep copy of the ReadOnlyAddressBook
```
Comment (Rishit02) [inline src/main/java/seedu/address/model/ReadOnlyAddressBook.java:19]: Full stop.

**Review by Rishit02 — APPROVED** [review]
LGTM.

**Review by Jolonauh — APPROVED** [review]
LGTM

**Pull request merged**